.github/workflows/ci-cd.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         config:
           - {os: ubuntu-20.04, r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
 
+    # we need to define a package manager for RStudio and expose the Github PAT for access
```

Comment:
"Define a package manager for RStudio and expose the Github PAT for access"

If you say "we need to ..." then you need to tell me why

Reply:
updated, @riccardoporreca  please let me know in case there is something inaccurate with the updated comment

Reply:
Doing this as part of the review comments